### PR TITLE
EE-2167 - Removed star2xml

### PR DESCRIPTION
**Pull request by M-casado**
Moved the star2xml tool to its own repo https://github.com/EGA-archive/star2xml/

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,27 +4,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+# [0.0.0] 
+## 2021-11-05
+### Modified
+- [star2xml](https://github.com/EGA-archive/star2xml) - Moved tool star2xml to its own repository: https://github.com/EGA-archive/star2xml. 
+- [JSON schemas](https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas) - improved drafted JSON schemas for AF (Array Format) Array and Experiment metadata .
```
Comment (malloryfreeberg) [inline CHANGELOG.md:11]: Should this be part of the PR for improving the AF JSON schemas instead?

**Review by malloryfreeberg — CHANGES_REQUESTED** [review]
Left one comment to check before approving.

File: CHANGELOG.md (hunk 1 above)
Comment (M-casado) [reply CHANGELOG.md]: Indeed, solved it and will put it in the other PR

**Review by malloryfreeberg — APPROVED** [review]
LGTM!

**Pull request merged**